### DISCUSSION TITLE
Controller test - how to disable HTML response to be logged in terminal ?

### DISCUSSION THREAD

**Discussion by Chaxwell** [symfony/symfony#49979] [Q&A]
Hey there, I would like to disable the logging of the HTML response when a test fails. At the moment it's really unreadable if you have more than one test failing :(
Ideally I'd like to get only the response headers and that's it.

I have tried disabling the catchExceptions flag on the KernelBrowser client as suggested [here](https://github.com/symfony/symfony/issues/40837) but that doesn't work.

I am using simple-phpunit.
Any idea please ?

Thanks

**Comment by elevado** [comment]
I think it cannot deactivated. This is what it looks like to me when I look at `assertResponseStatusCodeSame()` (`assertThatForResponse()`). 

For example, use `assertSame()`.

```php
$this->assertSame(Response::HTTP_OK, $client->getResponse()->getStatusCode());
//$this->assertResponseStatusCodeSame(Response::HTTP_OK);
```
Or:

```php
try {
    $this->assertResponseStatusCodeSame(Response::OK);
} catch (\Exception $e) {
    $this->fail(
        sprintf(
            'Failed asserting that the Response status code is %s.',
            $client->getResponse()->getStatusCode()
        )
    );
}
```

**Comment by gluisier** [comment]
At time of writing this, [since 7.1.0-BETA1](https://github.com/symfony/framework-bundle/commit/facab7ae1656d3217684dbbfc3ee4ee78f787229), [`assertResponseStatusCodeSame` has three parameters](https://github.com/symfony/framework-bundle/blob/7.3/Test/BrowserKitAssertionsTrait.php#L36), the last one being a boolean about verbosity, being **`true`** by default. Setting it to `false` will disable HTML content logging.
